Updating LDAP integration instructions

daviftorres opened the pull request:
Documentation page https://docs.cloudstack.apache.org/en/latest/adminguide/accounts.html
The instructions were transferred from bullet points to a table that organize in columns:

- OpenLDAP
  - Unchanged instructions from existent version.

- Active Directory
  - Since I just confirmed that ShapeBlues' instructions (https://www.shapeblue.com/using-cloudstack-4-3-with-microsoft-active-directory) are accurate, the existent was brought up to speed.

<!-- readthedocs-preview cloudstack-documentation start -->
----
📚 Documentation preview 📚: https://cloudstack-documentation--494.org.readthedocs.build/en/494/

<!-- readthedocs-preview cloudstack-documentation end -->

DaanHoogland commented:
thank you @daviftorres , I like your changes but there is a formatting issue with it. 
![image](https://github.com/user-attachments/assets/7cbde82b-949f-47c5-9c09-f3215b3dba28)


It might be solved outside of the actual text changes, but I do not know how. Any idea?

daviftorres commented:
hey @DaanHoogland , it is because the following strings are single word and there is no word-wrapping:

- `OU=APAC,DC=company,DC=com`
- `DC=company,DC=com`
- `CN=ACSGroup,DC=company,DC=com`
- `CN=ACSGroup,CN=Users,DC=company,DC=com`
- `CN=ACSServiceAccount,OU=APAC,DC=company,DC=com`
- `CN=ACSServiceAccount,CN=Users,DC=company,DC=com`

Do you think if we introduce a "\n" it will mislead the users to think it is not a single-line string?

daviftorres commented:
@DaanHoogland I found out there is no flag for wrapping long words but I know that LDAP accepts spaces after the commas. So, I changed my deployment and added the spaces everywhere and CloudStack continue to work as before. Example:
- `DC=company, DC=com`

DaanHoogland commented:
tnx @daviftorres

DaanHoogland commented:
btw @daviftorres , as this is based on main it will not reach the docs for the next dot-releases, only for v21. If you wish you can rebase it on 4.19 to have it in earlier.

davift commented:
All good!
Thank YOU for your hard excellent work keeping this project.
